spectrochempy/core/readers/read_spc.py

```diff
@@ -533,53 +533,49 @@ def _read_spc(*args, **kwargs):
     dataset = NDDataset(np.expand_dims(floatY, axis=0))
     dataset.name = str(filename)
     dataset.units = y_unit
-    dataset.title = y_title
-    dataset.origin = "thermo galactic"
+    dataset.long_name = y_title
+    dataset.source = "thermo galactic"
 
     # now add coordinates
     _y = Coord(
         [timestamp],
-        title="acquisition timestamp (GMT)",
+        long_name="acquisition timestamp (GMT)",
         units="s",
         labels=([acqdate], [filename]),
     )
 
     dataset.set_coordset(y=_y, x=_x)
 
-    dataset.description = kwargs.get("description", "Dataset from spc file.\n")
+    dataset.source = kwargs.get("description", "Dataset from spc file.\n")
     if ord(Fexper) != 0 and ord(Fexper) != 7:
-        dataset.description += "Instrumental Technique: " + technique + "\n"
```

Comment:
I would prefer that `technique`,  `resolution`,`instrument`,  `scmnt`, logtext, etc.. go to `comment`, not `source`.

I my view, `source` should focus on the (type of) source file and - when available -  who/which institution has produced the data ; `comment`  would group all other descriptive informations. This is probably because I have been influenced by the J-CAMP format, which defines `ORIGIN`, and `NOTES` as follows;

ORIGIN: Name of organization, address, telephone number, name of individual contributor, etc., as
appropriate. This information is not optional; the originator of a JCAMP-DX file should
always see that it is included.

The NOTES subset complement the CORE by providing information on the sample,
equipment, method or observation, data processing, etc. Experience has shown that the
needs of users span the full range from those who just want data in an ASCII file for input to their own software, to
those who want to be able to describe every experimental detail in the same file as the data.
By and large, the NOTES consist of information describing a sample, method of preparation
and observation, on instrumentation and operating conditions, and on data processing.
Instrument vendors are encouraged to include as much information is as availble from their
native format data files

I have no religion about that, but we should probably be a bit more specific in the NDDataset docsring as to the type of information that go in source and comment :-)

Reply:
No problem. I thought that Origin/ Source was just the name of the software producing the data: OPUS, OMNIC, TOPSPIN… but you can change this. We need to do this for all readers. Most of the other information can go in the comments (I don’t think we need to add another Note field).


And actually this is an error of string replacement, should be comment instead of source.